source/blog/2016-05-19-mux-shark.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Muxshark: A Wireshark plugin for the Mux protocol"
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Mux
+---
+
+This [Wireshark](https://www.wireshark.org/) dissector provides basic decoding of all
+[Mux](http://twitter.github.io/finagle/guide/Protocols.html#mux) messages including:
+
+* Frame length
+* Message type
+* Tag number
+
+> <sub>Don't know about [Wireshark](http://jvns.ca/blog/2016/03/16/tcpdump-is-amazing/)
+or [Mux](http://twitter.github.io/finagle/guide/Protocols.html#mux)?
+Follow the links for introductions.</sub>
+
+While this was released as part of Finagle 6.33.0 we never got around to properly announcing it. Now that Shark Week 2016 is about a month away, it felt like a good time to tell people about it.
```

Comment:
+1 for the shark week reference although i'm a bit disappointed there are no shark images in here.